tutorials/introductory/customizing.py

```diff
@@ -107,15 +107,24 @@
 # the matplotlib package. rcParams can be modified directly, for example:
 
 mpl.rcParams['lines.linewidth'] = 2
-mpl.rcParams['lines.color'] = 'r'
+mpl.rcParams['lines.linestyle'] = '--'
 plt.plot(data)
 
+###############################################################################
+# Note, that in order to change the usual `plot` color you have to change the 
+# `prop_cycle` property of `axes`:
```

Comment:
Since neither `prop_cycle`, nor `axes` are actual python objects, this line causes an error in the documentation built. You can use 

    [...] *prop_cycle* property of *axes*:

instead.